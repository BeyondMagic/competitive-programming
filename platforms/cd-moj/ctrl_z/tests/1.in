desfazer
inserir Era uma vez
inserir uma terra muito
inserir distante
desfazer
inserir em que tudo era muito legal.
inserir Mas ai, de
inserir repente
desfazer
desfazer
desfazer
