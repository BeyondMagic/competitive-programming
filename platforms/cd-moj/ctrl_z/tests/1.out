NULL
distante
repente
Mas ai, de
em que tudo era muito legal.
